source/reference/api/start.txt

```diff
@@ -320,21 +320,20 @@ When ``sharding.createSupportingIndexes`` is ``true``:
 
 * If the supporting indexes exist on the source cluster, ``mongosync`` 
   syncs the indexes to the destination cluster and uses them 
-  as the shard key.
+  as shard keys. 
 
 * If the supporting indexes don't exist, ``mongosync`` creates them on the
   destination cluster.
 
-The ``sharding.createSupportingIndexes`` option affects all collections sharded
-during sync.
+The ``sharding.createSupportingIndexes`` option affects all collections sharded.
```

Comment:
Nit, I would change `option affects all collections sharded.` to `option affects all sharded collections.`

Reply:
Fixed